src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -733,7 +743,7 @@ enum SetterConversion {
                 JDBCType.Category.TIME, JDBCType.Category.TIMESTAMP, JDBCType.Category.DATETIMEOFFSET,
                 JDBCType.Category.CHARACTER, JDBCType.Category.LONG_CHARACTER, JDBCType.Category.NCHARACTER,
                 JDBCType.Category.LONG_NCHARACTER, JDBCType.Category.BINARY, JDBCType.Category.LONG_BINARY,
-                JDBCType.Category.GUID, JDBCType.Category.SQL_VARIANT)),
+                JDBCType.Category.GUID, JDBCType.Category.SQL_VARIANT, JDBCType.Category.VECTOR)),
```

Comment:
(similar to above) Do we need JDBCType.Category.VECTOR in GetterConversion for the CHARACTER type, and vice versa?